conductr_cli/sandbox_stop_jvm.py

```diff
@@ -0,0 +1,84 @@
+def stop(args):
+    """
+    Stops the existing ConductR core and agent processes.
+    This is done by interrogating the output of the ps, looking for java process which is running of the sandbox image.
+    directory.
+
+    :param args: args parsed from the input arguments
+    """
+
+    def find_conductr_core_pids(run_dir):
```

Comment:
In `sandbox_run_jvm` the "local" functions that are called in the `run` function are not inner functions. In `sandbox_stop_jvm` they are inner functions. We should be consistent. I'd prefer to use the style as in `sandbox_run_jvm` because then you can see on the top what functions are called.